Makefile

```diff
@@ -74,7 +74,10 @@ clean:
 
 .PHONY: test
 test:
```

Comment:
I would prefer a separate target (perhaps `make test/coverage`?) for generating the test coverage profile. I believe `make test` should be reserved for just running the unit tests.

Reply:
Added new target as requested: `make test/coverage`

Reply:
Oh I just realized while `test/coverage` is in line with what we use to separate sub-commands of other make targets (e.g. `e2e/cluster`, the `test-sanity` target now sticks out. I think this is fine though (if anything we should consider changing `test-sanity` to `test/sanity` in a separate PR).